config/grumphp/hooks/commit-msg

```diff
@@ -24,19 +24,12 @@ DOCKER_PHP_CONTAINER_ID=$(docker-compose -f ${DOCKER_COMPOSE_FILE} ps -q phpfpm)
 # Remove single quotes from hook_command. It is needed to use it inside the docker exec
 HOOK_COMMAND=$(echo "$(HOOK_COMMAND)" | sed "s/'//g")
 
-printf "%s\n" "${DIFF}" > "${DIFF_FILE}" && docker cp "${DIFF_FILE}" ${DOCKER_PHP_CONTAINER_ID}:"/tmp/${DIFF_FILE}"
-
 # Run GrumPHP
-docker exec -t \
-    -e HOOK_EXEC_PATH=${HOOK_EXEC_PATH} -e HOOK_COMMAND="${HOOK_COMMAND}" \
-    -e GIT_USER="${GIT_USER}" -e GIT_EMAIL="${GIT_EMAIL}" -e COMMIT_MSG_FILE="${COMMIT_MSG_FILE}" \
-    -e DIFF_FILE="${DIFF_FILE}" \
-    ${DOCKER_PHP_CONTAINER_ID} sh -c \
-    'cd "${HOOK_EXEC_PATH}" && cat "/tmp/${DIFF_FILE}" | ${HOOK_COMMAND} --git-user="${GIT_USER}" --git-email="${GIT_EMAIL}" "${COMMIT_MSG_FILE}"'
-GRUMPHP_EXIT_CODE=$?
+(cd "./" && echo "${DIFF}" | \
+    exec 'docker' 'run' '--rm' '-t' "--volume=$PWD:${HOOK_EXEC_PATH}" "${DOCKER_PHP_CONTAINER_ID}" \
```

Comment:
Hi @ryantfowler this one is also failing for the same reason as #44:
```
docker: Error: No such image: 62a2c063b218ae30c8f8656487771e158671b5d5390791ac578341649d9eb662.
See 'docker run --help'.
```

I think that `docker run` expects an image identifier instead of a container id: `"${DOCKER_PHP_CONTAINER_ID}"`